app/internal/zodiac.py

```diff
@@ -0,0 +1,41 @@
+from datetime import date
+from typing import Dict, Union
+
+from sqlalchemy.orm import Session
+from sqlalchemy import and_, or_
+
+from app.database.models import Zodiac
+
+
+def create_zodiac_object(zodiac_fields: Dict[str, Union[str, int]]) -> Zodiac:
+    """This function create a zodiac object from given fields dictionary.
+    It is used for adding the data from the json into the db"""
+    return Zodiac(
+        name=zodiac_fields['name'],
+        start_month=zodiac_fields['start_month'],
+        start_day_in_month=zodiac_fields['start_day_in_month'],
+        end_month=zodiac_fields['end_month'],
+        end_day_in_month=zodiac_fields['end_day_in_month']
+        )
+
+
+def get_zodiac_of_day(session: Session, date: date) -> Zodiac:
+    """This function return a zodiac object
+    according to the current day."""
+    zodiac_obj = session.query(Zodiac).filter(
+        or_((and_(Zodiac.start_month == date.month,
```

Comment:
This one is a bit tricky. Use `_between` or export some expression to a variable with an indicative name :)